src/site/_drafts/what-should-you-measure-to-improve-performance/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: What should you measure to improve performance
+author: martinschierle
+description: |
+  Learn what to measure to optimize different parts of the funnel for performance and therefore conversions.
+web_lighthouse: N/A
+date: 2019-05-01
+---
+
+
+The different steps of a purchase funnel are prone to performance issues in
+different ways, and therefore need different measurement and optimizations:
+
+![image](funnel.png)
+
+In this guide we'll address how the different steps can be measured. For this we
+recommend you to look at lab as well as field data.
+
+**Lab data** is gathered by running tests locally, for example by using
+[Lighthouse](https://web.dev/fast/discover-performance-opportunities-with-lighthouse)
+and other tools.  This can make it possible  to compare website performance over
+time and with competitors through a controlled, stable environment, but it might
+not be representative of the performance users experience in real life.   
+
+**Field data** is gathered via analytics from real users, and is therefore
+representative of their experience. However, it can't easily be compared over
+time or with competitors. Network connections and smartphone hardware evolve
+over time, and different target audiences might have different devices, thereby
+making comparisons with field data tough.   
+
+To get a complete picture, both data sources are needed. The following sections
+show how to gather lab and field data for the different relevant performance
+marks across the funnel.
+
+## Discovery
+
+Optimizing for discovery means optimizing for the first load, which is what new
+users  will get, but also search and social
+[crawlers](https://developers.google.com/search/docs/guides/rendering). Improving on first load is
+covered in greater detail [here](https://web.dev/fast).   
+Lab data for a first load can be easily acquired through
+[Lighthouse](https://web.dev/fast/discover-performance-opportunities-with-lighthouse),
+while field data (at least for Chrome) is readily available through [Chrome UX
+reports](https://web.dev/fast/chrome-ux-report). A convenient combination of
+both can be found in [PageSpeed
+Insights](https://developers.google.com/speed/pagespeed/insights/). You should
+also track relevant metrics from the field yourself:
```

Comment:
The next three lines read a bit awkward to me. Are they variations of the same thing?

Reply:
Not sure if I understand this one - which three lines do you mean, the three metrics listed, because they are already part of lighthouse? Or do you mean the test highlighted here? The idea for the text was that the reader needs lab data (lighthouse), field data (measured themselves) and that both together can be obtained through PSI.